src/main/java/org/sagebionetworks/bridge/hibernate/HibernateStudy.java

```diff
@@ -247,7 +251,27 @@ public String getDetails() {
     public void setDetails(String details) {
         this.details = details;
     }
-    
+
+    @Override
+    public Exporter3Configuration getExporter3Configuration() {
+        return exporter3Configuration;
+    }
+
+    @Override
+    public void setExporter3Configuration(Exporter3Configuration exporter3Configuration) {
+        this.exporter3Configuration = exporter3Configuration;
+    }
+
+    @Override
+    public boolean isExporter3Enabled() {
+        return exporter3Enabled;
+    }
```

Comment:
Add an AuthUtils check in updateStudy so that only ORG_ADMIN can turn this on/off. (We can later refactor this as part of the permissions stuff Jack is working on.)